### ISSUE TITLE
Read Dead Redemption GOTY crash

### ISSUE THREAD

**Issue by morten-holm** [RPCS3/rpcs3#16288]
### Quick summary

Game crashes on start

### Attach a log file

[RPCS3.log.zip](https://github.com/user-attachments/files/17621330/RPCS3.log.zip)
[mac-crash.txt.gz](https://github.com/user-attachments/files/17621363/mac-crash.txt.gz)

### System configuration

Model: Mac14,3, BootROM 11881.41.5, proc 8:4:4 processors, 24 GB, SMC 
Graphics: Apple M2, Apple M2, Built-In

**Comment by CrazyCSIW6** [comment]
1. This issue isn't valid
2. Your hardware is terrible for this emulator
3. That title ID makes no sense

WTF am I reading??

**Comment by morten-holm** [comment]
> 1. This issue isn't valid
> 2. Your hardware is terrible for this emulator
> 3. That title ID makes no sense
> 
> WTF am I reading??

Just out of curiosity. 

1. What makes it invalid? Are you missing information?

2. I apologise for my 'terrible hardware'. Does it crash because of the hardware?

3. I changed the title.

**Comment by mavethee** [comment]
@CrazyCSIW6 No kurwa debil 🤦🏻 

That's my matching quality response on the issue.

Are you alright dude? since when github is a homie group chat? XD

Regardless,

@morten-holm

1. Before opening an issue, ensure you're using per-game settings from [RPCS3 Wiki](https://wiki.rpcs3.net/index.php?title=Red_Dead_Redemption), if available (in case of ingame games as RDR, they are available, use defaults only when said, they are just fine).

2. Ignore, that's a PCs are main culpit here in childlish way. Mac support is experimental, **keep that in mind**.

3. With titles, look at other issues, it's usually e.g: `Red Dead Redemption: Game of the Year Edition [BLES01294] - Issue XYZ`, same for system config, but that's minor.

TL;DR 

Follow RPCS3 Wiki for best settings, should be more stable, but again couple of issues, RDR isn't playable, expect issues and crashes regardless some settings that help with stability. For general support, use Discord server, here are reported general issues as emulator focuses on accuracy to the PS3, not fix exclusive game issues.

(ONLY EDIT WHEN IT'S ISSUE WITH COMPILATION ON BOOT) With macOS 15, I believe there's an issue with LLVM CPU and it freaks out during compilation on boot.

Open `config.yml` in your RPCS3 dir (default is `~/Library/Application Support/rpcs3`), locate `Use LLVM CPU` and it should be `westmere` or `"westmere"` not `Westmere`, if you're not using AVX exposure.

Here's "works for me" (very low digits as I was too lazy to close resources needy tasks):
<details>

![CleanShot 2024-11-06 at 08 42 14@2x](https://github.com/user-attachments/assets/1178ec3d-6ef7-435c-9493-67b0304d2ef3)

![CleanShot 2024-11-06 at 08 50 35@2x](https://github.com/user-attachments/assets/a71213f9-117c-42f6-a54e-6bd3450125a3)

</details>

## System Information:

- SYS: RPCS3 v0.0.34-17097-c68f42e0 Alpha | HEAD
- SYS: VirtualApple @ 2.50GHz | 8 Threads | 16.00 GiB RAM | TSC: 1.000GHz | FMA3
- SYS: Operating system: macOS, Version: 15.2.0
- SYS: Current Time: 2024-11-06T08:56:55
- RSX: Found Vulkan-compatible GPU: 'Apple M1' running on driver 0.2.995
- SYS: Firmware version: 4.91

## Log file:

[RPCS3.log.gz](https://github.com/user-attachments/files/17643566/RPCS3.log.gz)
